settings/settings.json

```
@@ -234,6 +234,13 @@
                         "type": "integer",
                         "default": "365"
                     },
+                    {
+                        "title": "Enable Rewatching Next Up",
+                        "description": "Enable showing already watched episodes in 'Next Up' sections.",
```

Remove `enable` to match the new title
```suggestion
                        "description": "Show already watched episodes in 'Next Up' sections.",
```